### DISCUSSION TITLE
Welcome to editor-frontend Discussions!

### DISCUSSION THREAD

**Discussion by NegiAkash890** [NegiAkash890/editor-frontend#10] [Announcements]
Discusssion can be done here
Status: I am reviewing all the PRs.

**Comment by arp99** [comment]
@NegiAkash890 My scores are not being updated for SWOC for the PR #8 and PR #26 , Please look into it, as not much mentors are active on discord.

**Comment by NegiAkash890** [comment]
Everyone is requested to list their Participant ID along with the MR(merge request number) number they have worked on so I can update the scores.

**Reply by arp99** [reply]
I am listing it here only, in some time

**Reply by arp99** [reply]
Participation Id: 174
Merged PR's
| Sl. no     | Merged PR No. |
| ----------- | ----------- |
| 1     | PR #8  |
| 2   | PR #26  |

**Reply by Rishi-121** [reply]
PID: 411

| Sl.No | Merged PR No. |
|-------|------------------|
| 01    | PR #29   |
| 02 | PR #31  |

Issue no #20 is also resolved with PR #29.  Do check! I have attached the screenshot of all the changes in PR #29.
